n3fit/src/evolven3fit/evolve.py

```diff
@@ -125,7 +133,9 @@ def _wrap_evolve(pdf, replica):
             evolved_blocks = evolve_exportgrid(pdf, eko_op, x_grid)
             dump_evolved_replica(evolved_blocks, usr_path, int(replica.removeprefix("replica_")))
 
-        Parallel(n_jobs=NUM_CORES)(
+        # Choose the number of cores to be the Minimal value
+        nb_cores = min(NUM_CORES, ncores)
```

Comment:
Since you put `1` as the default I think it is fine if you just accept what the user asks for.

Reply:
Right! I was mainly thinking in the situation in which someone is asking more than what's available 🙈

Reply:
Done in [a1f5abd](https://github.com/NNPDF/nnpdf/pull/2168/commits/a1f5abd0ed5a674369caa0682fa19e82402c6d4f). 

In the decided to keep the `min` as it doesn't make more sense to run more concurring jobs than the available `cores`.